src/actions/index.js

```diff
@@ -5,7 +5,7 @@ export const createBook = book => ({
   book,
 });
 
-export const removeBook = id => ({
+export const handleBookRemove = id => ({
```

Comment:
This should remain `removeBook`.
```suggestion
export const removeBook = id => ({
```